test/clj/webchange/test/fixtures/core.clj

```diff
@@ -102,18 +104,23 @@
            :version-id version-id)
           (transform-keys ->kebab-case-keyword)))))
 
-(defn scene-created []
-  (let [{course-id :id course-name :name course-slug :slug} (course-created)
+(defn scene-created
+  ([]
+   (let [course (course-created)]
+     (scene-created course)))
+  ([course]
+  (let [{course-id :id course-name :name course-slug :slug} course
         scene-name "test-scene"
         [{scene-id :id}] (db/create-scene! {:course_id course-id :name scene-name})
-        data {:test "test" :test-dash "test-dash-value" :test3 "test-3-value"}
```

Comment:
We need to keep this data as is. It MUST be :test3 because we have such keys in existing courses and scenes.
Please see webchange.db namespace.